silta/silta-prod.yml

```diff
@@ -64,3 +64,13 @@ mariadb:
 # Disable the built-in database when using an external database.
 #mariadb:
 #  enabled: false
+
+varnish:
+  enabled: true
+  resources:
```

Comment:
Should we add limits as well?

Reply:
Yes, it's good to provide `limits` values also as an example. There should also be a comment that they need to be adjusted per project.
But what would be good or reasonable example values @evilfurryone if the file `storageBackend` is used with `512M` below?

Reply:
> There should also be a comment that they need to be adjusted per project.

This applies to all services, not only Varnish.

Reply:
Increased, 086a2d8.